deltaG/code/run_dgpredictor.py

```diff
@@ -0,0 +1,403 @@
+import pandas as pd
+import numpy as np
+import re
+import json
+import joblib
+import sys
+
+# to your dGPredictor\CC path
+sys.path.append(r"D:\model_research\dGPredictor\CC")
```

Comment:
Can you also provide some instructions on what are software requirements to install etc?